endpoint/opcua/src/main/java/de/fraunhofer/iosb/ilt/faaast/service/endpoint/opcua/listener/AasServiceIoManagerListener.java

```diff
@@ -186,26 +108,9 @@ public boolean onWriteNonValue(ServiceContext sc, NodeId nodeid, UaNode uanode,
     }
 
 
-    /**
-     * Notification of a write request for the Value attribute of a Variable node. The method is called before write
-     * actually takes place. So you can cancel the write, for example, if the user is not allowed to write to the value.
-     *
-     * @param sc The serviceContext of the client connection used to call this service.
-     * @param nodeId The NodeId of node to write.
-     * @param uvn The node object to write. If the node is not available this may be null.
-     * @param indexRange The index range to set for an array value. May be null.
-     * @param dv The data value to write.
-     * @return true if the value was written to the source and you do not want any other operations to continue writing
-     *         it - including it being written to the node (if you are using an IoManagerUaNode). If you return false, other
-     *         listeners and the IoManager will get called with the write request.
-     * @throws StatusException If the write fails. Expected result codes: Bad_NodeIdInvalid, Bad_NodeIdUnknown,
-     *             Bad_AttributeIdInvalid, Bad_IndexRangeInvalid, Bad_IndexRangeNoData, Bad_DataEncodingInvalid,
-     *             Bad_DataEncodingUnsupported, Bad_NotWriteable, Bad_UserAccessDenied, Bad_OutOfRange, Bad_TypeMismatch,
-     *             Bad_WriteNotSupported
-     */
     @Override
     public boolean onWriteValue(ServiceContext sc, NodeId nodeId, UaValueNode uvn, NumericRange indexRange, DataValue dv) throws StatusException {
-        LOGGER.info(
+        LOGGER.trace(
                 "onWriteValue: nodeId={}{}{} value={}", nodeId, uvn != null ? " node=" + uvn.getBrowseName() : "", indexRange != null ? " indexRange=" + indexRange : "", dv);
 
         try {
```

Comment:
Move null-check of endpoint to constructor

Reply:
moved to constructor